src/main/java/com/wetwater/washwater/item/FluidPipetteItem.java

```diff
@@ -20,19 +20,16 @@ public InteractionResult useOn(UseOnContext useOnContext) {
         Player player = useOnContext.getPlayer();
         ItemStack itemStack = useOnContext.getItemInHand();
         BlockPos targetPos = useOnContext.getClickedPos();
-        if (!level.isClientSide) {
-            if (!player.isCrouching()) {
-                System.out.println("Placed Water with Pipette");
-                BucketMechanics.creativePipettePlace(level, targetPos, itemStack, player);
-            }
-            else {
-                System.out.println("Picked up Water with Pipette");
-                BucketMechanics.creativePipettePickup(level, targetPos, itemStack, player);
-            }
-
+        if (!player.isCrouching()) {
+            System.out.println("Placed Water with Pipette");
+            BucketMechanics.creativePipettePlace(level, targetPos, itemStack, player);
```

Comment:
So instead of calling BucketMechanics just put that method here. KEEP it in a seperate method

Reply:
amongus